lib/fluent/plugin/os.rb

```diff
@@ -45,4 +49,22 @@ def self.os_name_ubuntu?
     false
   end
 
+  def self.os_name_debian?
+    os_name = 'not_found'
+    file_name = '/etc/os-release'
+    if File.exists?(file_name)
+      File.foreach(file_name).each do |line|
+        if line.start_with?('ID=')
+          os_name = line.split('=')[1].strip
+        end
+      end
+    else
+      logger.info('Unknown linux distribution detected')
+    end
+    os_name == 'debian' ? true : false
+  rescue StandardError => e
+    log.error "Unable to detect ubuntu platform due to: #{e.message}"
```

Comment:
`s/ubuntu/debian/` ?

Reply:
i am not clear what you intend to say. Please elaborate

Reply:
Oh, this is a [sed](https://en.wikipedia.org/wiki/Sed) command that suggests to replace the first pattern with the next.